src/components/Icons/index.tsx

```diff
@@ -21,12 +22,13 @@ export const Loading = <LoadingOutlined style={{ fontSize: 40 }} spin />;
 export const Play = <CaretRightOutlined />;
 export const Pause = <PauseOutlined />;
 export const UploadFile = <ImportOutlined />;
-export const DownArrow = <DownOutlined />;
+export const DownArrow = <DownOutlined size={32} />;
 export const CaretRight = <CaretRightOutlined />;
 export const GoBack = <ArrowLeftOutlined />;
 export const Reset = <HomeOutlined />;
 export const ZoomIn = <PlusOutlined />;
 export const ZoomOut = <MinusOutlined />;
+export const Download = <DownloadOutlined size={32} />;
```

Comment:
this seems redundant with `DownArrow`?

Reply:
They are different icons, the download one has an arrow pointing into a tray.